python/bin/watchman-make

```diff
@@ -146,6 +159,14 @@ Define the root of the project.  The default is to use the PWD.
 All patterns are considered to be relative to this root, and the build
 tool is executed with this location set as its PWD.
 """)
+parser.add_argument('-e', '--exclude', type=str, nargs='+',
```

Comment:
can we change `-e` to `-x` or `-X`?  This is slightly more consistent with other unix tools and our own legacy simple pattern syntax (https://facebook.github.io/watchman/docs/simple-query.html).  I'm aware that the `-p` option above isn't consistent with the simple pattern syntax either, but it's not worth changing.

`-e` makes me think of executing rather than excluding.